src/core/dbus-manager.c

```diff
@@ -2495,6 +2495,7 @@ const sd_bus_vtable bus_manager_vtable[] = {
         SD_BUS_PROPERTY("DefaultTimerAccuracyUSec", "t", bus_property_get_usec, offsetof(Manager, default_timer_accuracy_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("DefaultTimeoutStartUSec", "t", bus_property_get_usec, offsetof(Manager, default_timeout_start_usec), SD_BUS_VTABLE_PROPERTY_CONST),
         SD_BUS_PROPERTY("DefaultTimeoutStopUSec", "t", bus_property_get_usec, offsetof(Manager, default_timeout_stop_usec), SD_BUS_VTABLE_PROPERTY_CONST),
+        SD_BUS_PROPERTY("DefaultTimeoutAbortUSec", "t", bus_property_get_usec, offsetof(Manager, default_timeout_abort_usec), SD_BUS_VTABLE_PROPERTY_CONST),
```

Comment:
i think it would be nicer if the bus property would actually be mapped to the same value if default_timeout_abort_usec is zero here. i.e. instead of passing out the internal variable 1:1 over thus bus I think a mapping function should return the value of default_timeout_stop_usec if set to zero, and only other the actual value of the variable.

Reply:
Makes sense. Fixed.